vowpalwabbit/rand48.cc

```diff
@@ -20,6 +20,12 @@ float merand48(uint64_t& initial) {
   return bits_to_float(temp) - 1;
 }
 
+float merand48(unsigned long& initial) {
```

Comment:
There was an issue posted that should be fixed in the master now.  Fix = convert size_t to uint64_t elsewhere.  This seems better because it promotes cross-platform compatibility and less code.